__tests__/app.test.js

```diff
@@ -272,3 +272,64 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("returns 200 status code and updates the article's votes when increasing vote count", () => {
+    const incVotes = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(incVotes)
+      .then((response) => {
+        const updatedArticle = response.body.article;
+        expect(response.status).toBe(200);
+        expect(updatedArticle).toHaveProperty(
+          "title",
+          "Living in the shadow of a great man"
+        );
+        expect(updatedArticle).toHaveProperty("topic", "mitch");
+        expect(updatedArticle).toHaveProperty("author", "butter_bridge");
+        expect(updatedArticle).toHaveProperty(
+          "body",
+          "I find this existence challenging"
+        );
+        expect(updatedArticle).toHaveProperty(
+          "created_at",
+          "2020-07-09T20:11:00.000Z"
+        );
+        expect(updatedArticle).toHaveProperty("votes", 101);
+        expect(updatedArticle).toHaveProperty(
+          "article_img_url",
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
+  test("returns updated article with article's votes when decreasing vote count", () => {
+    const decreaseVote = { inc_votes: -2 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(decreaseVote)
+      .then((response) => {
+        expect(response.body.article).toHaveProperty("votes", 98);
```

Comment:
Again .toMatchObject or objectContaining would be good here, as you could also check this is being decremented on the correct article id